lib/api.js

```diff
@@ -1,19 +1,20 @@
 const api = require('@asl/service/api');
 const db = require('@asl/schema');
-
 const { NotFoundError } = require('./errors');
-
 const errorHandler = require('./error-handler');
+const Keycloak = require('./helpers/keycloak');
 
 module.exports = settings => {
 
   const app = api(settings);
   const models = db(settings.db);
+  const keycloak = Keycloak(settings.auth);
 
   app.db = models;
 
   app.use((req, res, next) => {
     req.models = models;
+    req.keycloak = keycloak;
```

Comment:
It's hard to justify adding this on every request when it's only needed in a single endpoint.

Can we move it into the routers where it's needed?